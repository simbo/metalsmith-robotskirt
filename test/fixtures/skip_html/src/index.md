
<strong>Even</strong> <em>basic</em> <abbr>HTML</abbr> tags are <u>not</u> allowed.
