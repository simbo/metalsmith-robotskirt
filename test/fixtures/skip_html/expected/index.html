<p>Even basic HTML tags are not allowed.</p>
